src/main/kotlin/no/nav/cv/notifikasjon/StatusRepository.kt

```diff
@@ -81,17 +60,14 @@ class StatusRepository(
         .firstOrNull()
         ?: Status.nySession(aktorId)
 
-
+    // Should this sort be descending?
```

Comment:
Tror det er implisitt, men sikkert greit å legge det til så vi ikke lurer på det i fremtiden.

Reply:
Den er vel `ASC` implisitt? Meningen med det sorteringen var å unngå giftpiller, slik at om en gammel sak ble liggende ville det komme lengre og lengre bak i resultatene og til slutt falle utenfor.

Reply:
Stemmer, det er `ASC` implisitt. Det gir mening å ta de eldste først ja, husker å ha gjort noe liknende på pam-cv-api sin outbox pattern for å unngå samme problem dersom outboxen ble større enn batch størrelse. 

Jeg ser også at det er brukt `DESC` i `statusPerFodselsnummer`, er det av samme grunn?